core/tests/dispatch_integration_tests.rs

```diff
@@ -1482,95 +1320,93 @@ fn network_status_port_block_unblock_ip() {
 // A new batch up small-pings are then sent and replied to.
 fn remote_delivery_overflow_network_thread_buffers() {
     let mut buf_cfg = BufferConfig::default();
-    buf_cfg.chunk_size(1280);
-    buf_cfg.max_chunk_count(10);
+    let chunk_size = 1000;
```

Comment:
I mean...naming the variable the same way as the function you are setting it on, doesn't really add any new information here. It just wastes space...